AntistasiOfficial_1.9.Altis/Functions/fn_location.sqf

```diff
@@ -1,24 +1,16 @@
-params[ "_name", "_getClass", "_return" ];
+params ["_name", "_getClass"];
 
-_return = "";
-"
-_value = switch ( _getClass ) do
-    {
-    case true :
-        {
-        if ( getText( _x >> 'name') == _name ) then
-            {
-            _return = configName _x;
-            };
-        };
-    case false :
-        {
-        if ( configName _x == _name ) then
```

Comment:
( configName _x == _name ) wrong condition, in the mission this may working and may not working